client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -64,6 +64,26 @@ public class RssSparkConfig {
           .withDescription(
               "The memory spill switch triggered by Spark TaskMemoryManager, default value is false.");
 
+  public static final ConfigOption<Integer> RSS_PARTITION_REASSIGN_LOAD_BALANCE_SERVER_NUM =
+      ConfigOptions.key("rss.client.partitionReassign.loadBalanceServerNum")
```

Comment:
How about `rss.client.rebalance.partitionServerNum`?

Reply:
`partitionReassign` is camel style. Could you avoid it?

Reply:
How about `reassign` ?